### DISCUSSION TITLE
Loras______updating

### DISCUSSION THREAD

**Discussion by Ivcannons00420** [comfyanonymous/ComfyUI#3952] [General]
🐍 if you are like me you probably have hundreds of loras, all of my loras are from https://civitai.com my question is say in the folder loras can you right click open in terminal , yes you can now is there a away to update all them loras if they have say version v3, or v2 ? say if i was to write a python script using civitai api to do this would it work ?

**Comment by BrechtCorbeel** [comment]
Yes anything is possible with code if you know how to do it.

Smatrix has a library of CivitAI models with versions, you would have to create the same and create a function or class or something that can detect latest version and either prompt you to download and update or do that itself.

Figure out or write an application that can detect that or an addon that does that for you by using the CivitAi API in Comfy itself.
